Answers.md

```diff
@@ -0,0 +1,14 @@
+1. If you had to describe semantic HTML to the next cohort of students, what would you say?
+Semantic html simply refers to any HTML tag which is descriptive. For instance, anything wrapped in a <section> would obviously be described as a section within the code structure. Anything within an <address> tag would be address information. This is not only for developers to easily comprehend the code but also for SEO purposes. When one searches Google for an address, these search engines can easily extract the data they need, like from the <address> tag, and show you the results you're looking for.
```

Comment:
This is definitely a good way to think about semantic HTML, but keep in mind that it is referring to structure just as much as it is referring to specific tags that have a semantic meaning.

I highly recommend taking a read through this: https://html.com/semantic-markup/

One part that I particularly like:

**There are two different practices that must be put into place if we are going to write semantic markup:**
1. Semantic markup requires that HTML elements be used according to their intended purpose.
2. Semantic markup requires the separation of content and presentation.